CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0]
+### Added
+* Multi-entrypoint capabilities to plugin
+
+### Changed
+* Burst workflow name to `insar_tops_burst`
```

Comment:
```suggestion
* Burst workflow renamed to `insar_tops_burst` from `topsapp_burst`
```

Reply:
Although I am suggesting we drop the burst specific console script entry point and (maybe) provide a `hyp3_isce2` one.